### PR TITLE
Added SPIRIT token

### PR DESCRIPTION
**Pull request by IrishLatte19**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @IrishLatte19

File: dbt_subprojects/tokens/models/prices/fantom/prices_fantom_tokens.sql [hunk 1]
```diff
@@ -88,7 +88,8 @@ FROM
     ('polter-polterfinance-protocol-token', 'fantom', 'POLTER', 0x5c725631FD299703D0A74C23F89a55c6B9A0C52F, 18),
     ('equal-equalizer-dex', 'fantom', 'EQUAL', 0x3fd3a0c85b70754efc07ac9ac0cbbdce664865a6, 18),
     ('ftails-ftails', 'fantom', 'FTAILS', 0x5cf90b977c86415a53ce3b7be13b26f6abddfee2, 18),
-    ('fbux-fantom-money-market', 'fantom', 'FBUX', 0x1e2ea3f3209d66647f959cf00627107e079b870d, 18)
+   -- ('fbux-fantom-money-market', 'fantom', 'FBUX', 0x1e2ea3f3209d66647f959cf00627107e079b870d, 18),
```
Comment (jeff-dude) [inline dbt_subprojects/tokens/models/prices/fantom/prices_fantom_tokens.sql:91]: what's the context for removing this one?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: dbt_subprojects/tokens/models/prices/fantom/prices_fantom_tokens.sql (hunk 1 above)
Comment (IrishLatte19) [reply dbt_subprojects/tokens/models/prices/fantom/prices_fantom_tokens.sql:91]: This token became inactive on Coinpaprika so tests were failing. That's why I commented it.

Comment (jeff-dude) [reply dbt_subprojects/tokens/models/prices/fantom/prices_fantom_tokens.sql:91]: okay thank you

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**